testcases/gui-local-auth-chek.md

```diff
@@ -0,0 +1,96 @@
+# Running Selenium authentication test cases for DOP
+<b>tcid:</b> gaau01 <br>
+<b>name:</b> "Running Selenium authentication test cases for DOP"<br>
```

Comment:
"Running Selenium cluster test cases for DOP 
to Local Authentication Testcases Using Browser